src/main/scala/beam/sim/BeamMobsim.scala

```diff
@@ -205,13 +204,13 @@ class BeamMobsim @Inject()(
         private val numRideHailAgents = math.round(
           beamServices.beamConfig.beam.agentsim.numAgents.toDouble * beamServices.beamConfig.beam.agentsim.agents.rideHail.numDriversAsFractionOfPopulation
         )
-        private val rideHailVehicleType =
-          BeamVehicleUtils
-            .getVehicleTypeById(
-              beamServices.beamConfig.beam.agentsim.agents.rideHail.vehicleTypeId,
-              scenario.getVehicles.getVehicleTypes
-            )
-            .getOrElse(scenario.getVehicles.getVehicleTypes.get(Id.create("1", classOf[VehicleType])))
+//        private val rideHailVehicleType =
```

Comment:
I'll have to check by pulling into an editor instead of using the github.com reviewer....but what is accomplished by commenting the val? The code will still execute and now just not be assigned to a variable. Should this be the whole code block?